build/config/webpack.main.js

```diff
@@ -33,6 +33,11 @@ export default {
       LIBDIR: 'require("path").join(__dirname, "..")',
     }),
   ],
+  resolve: {
+    alias: {
+      'tofino-log-utils': path.join(SRC_DIR, 'log-utils.js'),
```

Comment:
Appended 'tofino-' to the module just as an indication it may be aliased. Flexible with other idioms if you have any ideas